spec/jobs/segment_identify_job_spec.rb

```diff
@@ -37,10 +37,10 @@
 
       it 'includes hosting type equal to cloud' do
         expect(SEGMENT_CLIENT).to receive(:identify).with(
-          hash_including(traits: hash_including(hosting_type: 'cloud'))
+          hash_including(traits: hash_including(hosting_type: 'cloud')),
         )
 
-        subject.perform_now(membership_id: membership_id)
+        subject.perform_now(membership_id:)
```

Comment:
RSpec/NamedSubject: Name your test subject if you need to reference it explicitly. (https://rspec.rubystyle.guide/#use-subject, https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/NamedSubject)